scripts/verifyTestCourseJsonFiles.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+yarn exportAllCourses -B
```

Comment:
```suggestion
yarn exportAllCourses
```
 Or what was that about?